exporters/variables-scss/src/content/generators/fileGenerator.ts

```diff
@@ -0,0 +1,56 @@
+import { TokenGroup, Token, TokenType } from '@supernovaio/sdk-exporters';
+import { generateFileContent } from './contentGenerator';
+
+const filesData = [
```

Comment:
If this is some kind of configuration for the exporter, I suggest moving it to some directory like `constants` or having a `config.ts` file from which you can use it.